modules/i18n/src/translations/en-US/portals/actions.ts

```diff
@@ -148,7 +148,7 @@ export const actions: actionsNS = {
             validations: {
                 empty: "Endpoint is a required field.",
                 invalidUrl: "Please enter a valid URL.",
-                notHttps: "The entered URL is not HTTPS. Please add a valid URL."
+                notHttps: "The URL is not secure (HTTP). Use HTTPS for a secure connection."
```

Comment:
How about something like this? 

Http urls are not secure. Use https for a secure connection

Reply:
Let's go with this at this moment. Will refactor later.